lib/inc/puppetdb-cli/puppetdb-cli.hpp

```diff
@@ -109,7 +109,8 @@ LIBPUPPETDB_EXPORT PuppetDBConn get_puppetdb(const std::string& config_path,
  * @return This function does not return anything.
  */
 LIBPUPPETDB_EXPORT void pdb_query(const PuppetDBConn& conn,
-                                  const std::string& query);
+                                  const std::string& query,
+                                  const bool& is_pretty);
```

Comment:
Right now we don't use it per se, but we could generate the docs from it and use them :)